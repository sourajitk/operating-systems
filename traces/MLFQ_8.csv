1,10,100
2,20,90
3,30,80
4,40,70
5,50,60
6,60,50
7,70,40
8,80,30
9,90,20
10,100,15
11,110,10
12,120,5
13,130,10
14,140,15
15,150,20
16,160,25
17,170,30
18,180,35
19,190,40
20,200,45