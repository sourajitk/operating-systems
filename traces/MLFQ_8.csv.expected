1,800
2,790
3,770
4,740
5,700
6,650
7,583
8,486
9,359
10,283
11,120
12,125
13,140
14,284
15,360
16,426
17,487
18,538
19,584
20,620
